### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by HVRVKVT** [HDInnovations/UNIT3D#293]
# Add a wiki

It would be really nice to have a wiki for easy maintainable and searchable guides, faq, knowledge base, etc.
While the forum is great and superier for discussion and support, the format is not really suited for garthering information.

Best of all there is already an open source PHP based wiki available.
https://www.dokuwiki.org/
https://github.com/splitbrain/dokuwiki

Great features like:
- Index-based fast fulltext search
- Pagecaching
- Automatic table of contents generation
- Access Control Lists & Read only pages
- Unlimited page revisions  
- Colored side by side diff support
- Section Editing (allows editing of small parts of a page)
- Locking to avoid edit conflicts
- Automatic save to avoid content lost during editing
- Access Control Lists & Read only pages
- Plugin and theme support

More features  https://www.dokuwiki.org/features

**Comment by HDVinnie** [comment]
Unfourtionatly the one you linked is not Laravel based.

**Comment by HVRVKVT** [comment]
Going into uncharted waters but, Isn't Laravel just a framework for PHP, shouldn't it be deployable?

**Comment by InnocentPuff** [comment]
I think it could work in the end since PHP is being used but it would make the app alot more complex in to develop and maintain.

**Comment by HDVinnie** [comment]
This is correct....its possible but would be unwise to mix a pure php project dokuwiki into UNIT3D which uses Laravel a MVC framework.....work just make things messy IMO.
